Game/SpriteHandlers/SpriteFactory/ParticleSpriteFactory.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using MainGame.SpriteHandlers.ItemSprites;
+
+namespace MainGame.SpriteHandlers
+{
+    public enum ParticleSpriteTypes
+    {
+
+    }
+
+    public static partial class SpriteFactory
+    {
+
+
+        public static ISprite CreateDeathParticles()
+        {
+
+
+            return new ParticleSprites.DeathParticles(
+                TextureMap["ParticleSprites"], SpriteBatch,
+                numRows: 1,
+                numColumns: 2,
+                numberOfFrames: 2,
+                frameHeight: 16,
+                frameWidth: 16,
+                scale: UniversalScaleMultiplier) ;
+        }
+
+
+
+        public static ISprite CreateSwordBeamParticles()
+        {
+            return new ParticleSprites.SwordBeamParticles(
+                TextureMap["ParticleSprites"], SpriteBatch,
+                numRows: 1,
+                numColumns: 4,
+                numberOfFrames: 4,
+                textureStartingY: 18,               
```

Comment:
Generally, I think most of our sprites are 16x16, if they're not, then they're a multiple of 8. You can make this 16 and remove the frameHeight argument.